### PR TITLE
Some more fixes

### PR DESCRIPTION
**Pull request by hgschmie**
- allow "push" to be configured in the config file
- fix "invalid cookie" warning
- remove repeat of the "Wait for you to respond on your device" message
  for push
- allow 'cf' shortcut for changing factor
- allow 'nc' shortcut for new sms code
- remove unused okta library
- update aws library

### CODE REVIEW

File: src/main/java/com/okta/tools/AwsCli.java [hunk 1]
```diff
@@ -1019,7 +1022,7 @@ private static String mfa(JSONObject authResponse, boolean changeFactor) {
                 case ("question"): {
                     //question factor handler
                     String sessionToken = questionFactor(factor, stateToken);
-                    if (sessionToken.equals("change factor")) {
+                    if (sessionToken.equals("change factor") || sessionToken.equals("cf")) {
```
Comment (choang) [inline src/main/java/com/okta/tools/AwsCli.java]: lots of this expression gets repeated.  Time to consider cleaning up the code:
* make the support strings a `HashSet`
* change to `SUPPORTED.contians(sessionToken)`

I know you didn't name `sessionToken`, however, it seems like a misnomer unless I am misunderstanding the intent.

Comment (hgschmie) [reply src/main/java/com/okta/tools/AwsCli.java]: Currently, I am only patching this code. It is too ugly to start refactoring it. It is a hackathon project of mine to rewrite it in golang or rust.

Comment (hgschmie) [reply src/main/java/com/okta/tools/AwsCli.java]: why would I make this a hash set?

Comment (choang) [reply src/main/java/com/okta/tools/AwsCli.java]: maybe personal preference?  I think this looks cleaner:
```
if (SUPPORTED.contains(command)) {
...
}
```
than:
```
if (command.equals(string1) || command.equals(string2)) {
...
}
```

Comment (hgschmie) [reply src/main/java/com/okta/tools/AwsCli.java]: sure. I am simply not willing to rewrite the tool. ;-)

**Comment by hgschmie** [comment]
pretty ping?

**Pull request merged**